docs/datatable-tutorial/add_interactivity.md

```diff
@@ -0,0 +1,241 @@
+```python exec
+import reflex as rx
+from pcweb.base_state import State
+from pcweb.templates.docpage import docdemobox
+from pcweb.pages.docs.datatable_tutorial.states import DataTableState, DataTableState2
+
+```
+
+
+
+
+
+## Adding Interactivity to our DataTable
+
+
+Now we will add interactivity to our datatable. We do this using event handlers and event triggers.
+
+
+For our first interactivity example we will use the `on_cell_clicked` event trigger, this is called when the user clicks on a cell of the data editor. The event trigger receives the coordinates of the cell.
+
+```python
+class DataTableState(rx.State):
+    clicked_cell: str = "Cell clicked: "
+
+    ...
+
+
+    def get_clicked_data(self, pos) -> str:
```

Comment:
```suggestion
    def get_clicked_data(self, pos: tuple[int, int]) -> str:
```

Best to add an annotation here for readability, otherwise i have no idea what this `pos` object looks like

Reply:
added